project/LIS2DH12.c

```diff
@@ -64,15 +64,11 @@ typedef struct {
 															VARIABLES AND CONSTANTS
 *******************************************************************************/
 
-/*NOTE: Uncomment these lines for more debug info*/
-//#define SPI_DEBUG_INFO
-//#define ACCEL_DEBUG_INFO
-
 #define SPI_INSTANCE  0 //SPI instance index
 const nrf_drv_spi_t spi = NRF_DRV_SPI_INSTANCE(SPI_INSTANCE); //SPI instance
 volatile bool spi_xfer_done; //Flag used to indicate that SPI instance completed the transfer
 
-#define SPI_BUFFER_LENGTH (128)
+#define SPI_BUFFER_LENGTH (192)
```

Comment:
How'd you get this number?  If it's an EXACT number, maybe add 10% or round to the nearest 2^x so we have room for error and unexpected situations.